cmake/modules/RootBuildOptions.cmake

```diff
@@ -117,6 +117,7 @@ ROOT_BUILD_OPTION(cuda OFF "Enable support for CUDA (requires CUDA toolkit >= 7.
 ROOT_BUILD_OPTION(cudnn ON "Enable support for cuDNN (default when Cuda is enabled)")
 ROOT_BUILD_OPTION(cxxmodules OFF "Enable support for C++ modules")
 ROOT_BUILD_OPTION(dataframe ON "Enable ROOT RDataFrame")
+ROOT_BUILD_OPTION(dataframe_distributed_pyspark OFF "Enable distributed RDataFrame pyspark python module")
```

Comment:
I think it could be a bit long as a build option? (I imagine I will not be happy to type it. :)

Reply:
``` dataframe_distpyspark``?

Reply:
Sounds good, thanks!